processing.py

```diff
@@ -216,14 +223,15 @@ def basecall(blast_output_file, fastq_file, output_dir, quality_threshold, mode,
     base_filename = os.path.basename(fastq_file)
     alignments, ignored_reads, suspicious_reads, read_counter = get_alignments(blast_output_file, fastq_file=fastq_file,
```

Comment:
why not check if blast_output_file is empty here once instead of multiple checks and creating empty dfs..?

Reply:
the bug occurred when one of the individual blast files was empty – in the calculation per blast output.

Reply:
fixed